local/e2e/compose/compose_test.go

```diff
@@ -50,11 +50,11 @@ func TestLocalComposeUp(t *testing.T) {
 	const projectName = "compose-e2e-demo"
 
 	t.Run("up", func(t *testing.T) {
-		c.RunDockerCmd("compose", "up", "-d", "-f", "./fixtures/sentences/docker-compose.yaml", "--project-name", projectName, "-d")
+		c.RunDockerCmd("compose", "-f", "./fixtures/sentences/compose.yaml", "--project-name", projectName, "up", "-d")
```

Comment:
This change was made just to check we can put `-f` before or after `up`, right ? not because something is not supported anymore. Rebuilt and tester, seems OK, we still do support -f in both places

Reply:
I didn't tested we support `up -f` as I wanted to align with docker-compose which doesn't, but more flexibility doesn't hurt